frontend/lib/evictionfree/declaration-builder/confirmation.tsx

```diff
@@ -1,17 +1,151 @@
+const SAMPLE_USPS_TRACKING_NUMBER = "129837127326123";
+
+const HCA_HOTLINE_PHONE_LINK = "tel:+12129624795";
+const NYC_311_CONTACT_LINK =
+  "https://portal.311.nyc.gov/article/?kanumber=KA-02498";
+
+const LIST_OF_ORGANIZING_GROUPS_URL =
+  "https://d3n8a8pro7vhmx.cloudfront.net/righttocounselnyc/pages/1232/attachments/original/1590279936/List_of_Tenant_Organizing_Groups_Across_NY_State.pdf?1590279936";
+
+const checkCircleSvg = require("../../svg/check-circle-solid.svg") as JSX.Element;
+
+const TitleWithSymbol = () => (
+  <div className="media">
+    <div className="media-left">
+      <i className="has-text-info">{checkCircleSvg}</i>
+    </div>
+    <div className="media-content">
+      <h2 className="title is-size-4-mobile">
+        You've sent your hardship declaration
+      </h2>
+    </div>
+  </div>
+);
+
+const RetaliationBlurb = () => (
+  <>
+    <h2 className="title is-spaced has-text-info">
+      Contact a lawyer if your landlord retaliates
+    </h2>
+    <p>
+      It’s possible that your landlord will retaliate once they’ve received your
+      letter. This is illegal. Contact the City's Tenant Helpline (which can
+      provide free advice and legal counsel to tenants) by{" "}
+      <OutboundLink
+        href={`${NYC_311_CONTACT_LINK}`}
+        target="_blank"
+        rel="noopener noreferrer"
+        className="jf-nowrap"
+      >
+        calling 311
+      </OutboundLink>
+      .
+    </p>
+  </>
+);
+
+const HcaHotlineBlurb = () => (
+  <>
+    {" "}
+    <h2 className="title is-spaced has-text-info">Need additional support?</h2>
+    <p>
+      Call the Housing Court Answers hotline at{" "}
+      <OutboundLink
+        href={`${HCA_HOTLINE_PHONE_LINK}`}
+        target="_blank"
+        rel="noopener noreferrer"
+        className="jf-nowrap"
+      >
+        212-962-4795
+      </OutboundLink>
+      .
+    </p>
+    <p>
+      Hours of operation: Monday to Friday, 9am - 5pm. Available in English and
+      Spanish.
+    </p>
+  </>
+);
+
+const OrganizingGroupsBlurb = () => (
+  <>
+    {" "}
+    <h2 className="title is-spaced has-text-info">
+      Get involved in your local community organization and join the fight to
+      cancel rent.
+    </h2>
+    <p>
+      Join millions in the fight for a future free from debt and to win a
+      cancelation of rent, mortgage and utility payments.
+    </p>
+    <OutboundLink
+      className="button is-primary is-large jf-is-extra-wide"
+      href={LIST_OF_ORGANIZING_GROUPS_URL}
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      Find Organizations Near You →
+    </OutboundLink>
+    <p className="is-size-6">
+      <br />
+      *Due to the COVID-19 pandemic, some offices are closed and may not answer
+      phones.
+    </p>
+  </>
+);
 
 export const EvictionFreeDbConfirmation: React.FC<ProgressStepProps> = (
   props
 ) => {
+  const { session } = useContext(AppContext);
+  const isInNyc = session.norentScaffolding?.isCityInNyc;
```

Comment:
Err, definitely don't use this variable--the `norentScaffolding` is only populated during onboarding, meaning that if a pre-existing user from NYC has logged in, this value will be falsy even though the user is from NYC.

A better way of checking this is to see if `session.onboardingInfo.borough` is falsy: if it is, that means the user isn't in NYC.

Reply:
Removing this for now in prep for the demo tomorrow!